test/upload_test_helpers.js

```diff
@@ -92,11 +98,14 @@ module.exports = {
 
   cleanOutputDirectory() {
     deleteFolderRecursive(OUTPUT_PATH);
+	deleteFolderRecursive(OUTPUT_PATH_ALT);
   },
 
-  createOutputPath() {
-    if (!fs.existsSync(OUTPUT_PATH)) {
-      fs.mkdirSync(OUTPUT_PATH);
+  createOutputPath(outputPath) {
```

Comment:
You can set default values in the following way `createOutputPath(output = OUTPUT_PATH)`

Reply:
Switched to param defaulting